flask_session/sessions.py

```diff
@@ -160,6 +160,7 @@ def save_session(self, app, session, response):
 
         httponly = self.get_cookie_httponly(app)
         secure = self.get_cookie_secure(app)
+        samesite = self.get_cookie_samesite(app)
```

Comment:
Where is the definition of ``get_cookie_samesite()``?

Reply:
Its implemented in flask.sessions.SessionInterface of main Flask repository

Reply:
I see. But it doesn't look like anyone is maintaining the flask-session repo? Last commit was in 2017 as far as I can see

Reply:
Yea that's what I was worried about, I may just use my own fork. The feature is not critical for the project I'm working on, just nice to have.

Reply:
Yes, would be really nice to have it